acceptance/tests/identity/specify_unicode_user_ident.rb

```diff
@@ -4,15 +4,15 @@
 
 #Globals
 target_parent = 'c:/temp'
-target = 'c:/temp/specify_unicode_user_ident.txt'
-user_id = "user_렝딴슫있처"
+prefix = SecureRandom.uuid.to_s
+target = "c:/temp/#{prefix}.txt"
+raw_user_id = 'user_\uB81D\uB534\uC2AB\uC788\uCC98'
+user_id =     "user_\uB81D\uB534\uC2AB\uC788\uCC98" # 렝딴슫있처
 
 file_content = 'Flying Spaghetti Monster wants to save your soul.'
 verify_content_command = "cat /cygdrive/c/temp/specify_unicode_user_ident.txt"
-file_content_regex = /\A#{file_content}\z/
 
-verify_acl_command = "icacls #{target}"
-acl_regex = /.*\\user_렝딴슫있처:\(F\)/
+verify_owner_command = "(Get-ACL '#{target}' | ForEach-Object { $_.Access } | Where-Object { $_.IdentityReference -match ('\\\\' + [regex]::Unescape(\"#{raw_user_id}\")) -and $_.FileSystemRights -eq 'FullControl' } | Measure-Object).Count"
```

Comment:
This is checking `$_.Access` not `$_.Owner` like the other calls to `verify_owner_command`

Reply:
Copy-pasta fail.  Will fix.